server/service/src/sync/central_data_synchroniser.rs

```diff
@@ -77,7 +59,7 @@ impl CentralDataSynchroniser {
                 .sync_api_v5
                 .get_central_records(cursor, BATCH_SIZE)
                 .await
-                .map_err(|source| CentralSyncError::PullCentralSyncRecordsError { source })?;
+                .map_err(Error::PullCentralSyncRecordsError)?;
```

Comment:
removed source because I think this is cleaner, but still need to check how errors are displayed, downstream branch

Reply:
I've changed it again upstream, in the latest PR it seems to capture all errors pretty well